AcceleratorSampleApp/app/src/main/java/com/tokbox/android/accelerator/sample/ui/ActionBarFragment.java

```diff
@@ -213,16 +212,16 @@ public void updateCall() {
             mControlCallbacks.onCall();
     }
 
-    private void updateScreensharing() {
-        mVideoBtn.setOnClickListener(!mActivity.isScreensharing()
+    private void updateScreenSharing() {
```

Comment:
I have noticed different access modifiers for the same access in the anonymous class

- updateLocalAudio() - public void 
- updateLocalVideo() - public void 
- updateCall() - public void 
- updateScreensharing() - private void 
- updateAnnotations()  - private void 
- updateTextChat()  - private void 

If there is no reason for them to be like that, i believe they should all have same access.

Reply:
Some of them are used directly in the `MainActivity` and thus they have to be `public`. We could align access modifiers and make all of them public, but exposing more then is needed is a bad practice.

Reply:
I have changed few of them (where it was possible)
https://github.com/opentok/accelerator-sample-apps-android/pull/23/commits/7920502af2a916881d10debb9bcacc7cfa2c06b1

I think we should keep rest as it is

Reply:
I just checked main activity and found no usages for public methods below: 

updateLocalAudio() - public void
updateLocalVideo() - public void
updateCall() - public void

Reply:
made them private